tests/monotouch-test/Foundation/DateTest.cs

```diff
@@ -72,8 +71,20 @@ public void Precision32022 ()
 		{
 			var a = NSDate.Now;
 			var b = a.SecondsSinceReferenceDate - ((NSDate) (DateTime) a).SecondsSinceReferenceDate;
-			// ensure decimals are not truncated - but there's an unavoidable loss of precision
-			Assert.AreEqual (b, 0, 0.00001, "1");
+
+			// ensure decimals are not truncated - but there's an unavoidable loss of precision from ns to ms
+			Assert.AreEqual (b, 0, 0.001, "1");
+		}
+
+		[TestCase (1, 1, 1, 1, 1, 1, 1)]
+		[TestCase (199, 1, 1, 1, 1, 1, 1)]
+		[TestCase (2019, 1, 1, 1, 1, 1, 1)]
+		public void DateTimeToNSDate (int y, int m, int d, int h, int min, int s, int ms)
+		{
+			var date = new DateTime (y, m, d, h, min, s, ms);
+			date = DateTime.SpecifyKind (date, DateTimeKind.Utc);
```

Comment:
minor: there's a actor that let you specify the kind
https://docs.microsoft.com/en-us/dotnet/api/system.datetime.-ctor?view=netframework-4.8#System_DateTime__ctor_System_Int32_System_Int32_System_Int32_System_Int32_System_Int32_System_Int32_System_Int32_System_DateTimeKind_